chat-android/src/test/java/com/ably/chat/Sandbox.kt

```diff
@@ -0,0 +1,45 @@
+package com.ably.chat
+
+import com.google.gson.JsonElement
+import com.google.gson.JsonParser
+import io.ktor.client.HttpClient
+import io.ktor.client.engine.cio.CIO
+import io.ktor.client.request.delete
+import io.ktor.client.request.get
+import io.ktor.client.request.post
+import io.ktor.client.request.setBody
+import io.ktor.client.statement.HttpResponse
+import io.ktor.client.statement.bodyAsText
+import io.ktor.http.ContentType
+import io.ktor.http.contentType
+
+val client = HttpClient(CIO)
+
+class Sandbox {
```

Comment:
Perhaps not urgent, but would definitely recommend a setting/mode that can work with a local development realtime environment, will make future debugging a lot easier